jobs/go/sdk/dapr.yaml

```diff
@@ -1,4 +1,8 @@
 version: 1
+common:
+   daprdLogDestination: console
+   logLevel: info
+   schedulerHostAddress: localhost:50006
```

Comment:
will this static address of 50006 be resilient?

Reply:
Removed, although this could well fail - IIRC this was needed for the SDK to validate. If it fails I'll revert